frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/networking-review.tsx

```diff
@@ -1,34 +1,48 @@
 import * as React from 'react';
-import * as _ from 'lodash';
 import { connect } from 'react-redux';
 import { VMWizardNetwork } from '../../types';
 import { getNetworks } from '../../selectors/selectors';
-import { ReviewList } from './review-list';
 import { NetworkInterfaceWrapper } from '../../../../k8s/wrapper/vm/network-interface-wrapper';
 import { NetworkWrapper } from '../../../../k8s/wrapper/vm/network-wrapper';
+import { Table, TableBody, TableHeader, TableVariant } from '@patternfly/react-table';
 
 const NetworkingReviewConnected: React.FC<NetworkingTabComponentProps> = ({
   networks,
   className,
-}) => (
-  <ReviewList
-    title="Network Interfaces"
-    className={className}
-    items={networks.map(({ id, networkInterface, network }) => {
-      const networkInterfaceWrapper = new NetworkInterfaceWrapper(networkInterface);
-      const networkWrapper = new NetworkWrapper(network);
-      return {
-        id,
-        value: _.compact([
-          networkInterfaceWrapper.getName(),
-          networkInterfaceWrapper.getReadableModel(),
-          networkWrapper.getReadableName(),
-          networkInterfaceWrapper.getMACAddress(),
-        ]).join(' - '),
-      };
-    })}
-  />
-);
+}) => {
+  const headers = [
+    { title: 'Name' },
+    { title: 'Model' },
+    { title: 'MAC Address' },
+    { title: 'Network' },
+  ];
+
+  const rows = networks.map(({ networkInterface, network }) => {
+    const networkInterfaceWrapper = new NetworkInterfaceWrapper(networkInterface);
+    const networkWrapper = new NetworkWrapper(network);
+
+    return [
+      networkInterfaceWrapper.getName(),
+      networkInterfaceWrapper.getReadableModel(),
+      networkInterfaceWrapper.getMACAddress(),
+      networkWrapper.getReadableName(),
+    ];
+  });
+
+  return (
```

Comment:
we should use EmptyState when there are no networks. Please see `networking-tab.tsx`

\+ same for storage

Reply:
Done, but the styling could use some tweaking. The text looks odd being centered while everything else is left-aligned. It will probably look fine on small screens, but no so much on regular/large screens. Thoughts? @matthewcarleton 

![Selection_660](https://user-images.githubusercontent.com/8544299/79584775-db41d680-809c-11ea-8573-c6444339833b.png)

Reply:
ya, I'd suggest left aligning and just using body text. Although it feels like with not NICs there should be an alert of some kind. WDYT @suomiy ?

Reply:
Here's how it looks after the changes I made. Thoughts? @matthewcarleton @suomiy 
![Selection_661](https://user-images.githubusercontent.com/8544299/79603109-b576fa80-80b9-11ea-96a3-c0ee3b42cb01.png)

Reply:
Not sure, about alerts. It is a bit unusual situation, but the user should be quite aware what is going on - if he has VM without NICs for some reason (maybe will add them later)

Reply:
also, isn't it enough to say `No network interfaces`? @matthewcarleton @pcbailey